.github/workflows/validate-pr.yml

```diff
@@ -19,14 +19,34 @@ jobs:
         with:
           node-version: 12
 
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+
       - name: Cache dependencies
         uses: actions/cache@v2
         with:
-          path: /usr/local/share/.cache/yarn/v4
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
           key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
           restore-keys: |
             ${{ runner.os }}-yarn-
 
+      - name: Gatsby cache folder
```

Comment:
[nit] Can we rename this step (and the following) to have "Cache" as the first word? It seems like we're going with verb-first step names.

```suggestion
      - name: Cache Gatsby folder
```

Reply:
Haha I originally had something like that, but it read as `Cache Gatsby cache folder`. The double "cache" made me change it to what it is. There really isn't a "gatsby" folder per-se. Are you ok with `Cache Gatsby cache folder`?

Reply:
You can never have too much `cache`
![](https://media.giphy.com/media/h0MTqLyvgG0Ss/giphy.gif)